docs/endaq/quickstart.rst

```diff
@@ -80,3 +82,115 @@ information about the device and how it was configured.
   >>> with openFile('test.ide') as doc:
   ...     virtual_dev = endaq.device.fromRecording(doc)
 
+Quick Start Example Code
+========================
+Here is some starter code for introducing yourself to the endaq.device library. 
+Use it to familiarize yourself with with how to use endaq.device and learn 
+about some of its basic applications. Make sure to follow the 
+`installation <index.html#installation>` instructions found on the homepage 
+first. 
```

Comment:
Since this is documentation for a package on PyPI, I don't think the "Make sure to follow the installation instructions" sentence is necessary